editor/src/lib.rs

```diff
@@ -12,3 +12,9 @@ pub mod dispatcher;
 pub mod messages;
 pub mod test_utils;
 pub mod utility_traits;
+
+#[cfg(test)]
+#[test]
+fn export_types() {
+	specta::export::ts("../types.ts");
```

Comment:
Do we want to `#[ignore]` this text or is it fast enough?

Also we perhaps want to clearly document that you can run this test to regenerate the typescript.

Reply:
I think it's ok to keep around, it takes almost no time on my machine. I'll add some documentation for it.
<img width="340" alt="image" src="https://user-images.githubusercontent.com/14191578/211202871-f931f013-2aa7-4b1d-b761-8a92881e7896.png">